src/carma-streets/parse_ros_bags.py

```diff
@@ -0,0 +1,57 @@
+import rosbag
+import csv
+import pandas as pd
+import matplotlib.pyplot as plt
+import argparse
+from pathlib import Path
+
+
+
+def parse_ros_bag_file(ros_bag_file, csv_dir):
+    ros_bag_file_path = Path(ros_bag_file)
+    csv_dir_path = Path(csv_dir)
+
+# calculate the offset between ros ckock and the sim clock
+def get_time_offset(ros_bag_file):
+    sim_time_topic_name = '/sim_clock'
```

Comment:
We don't need a separate variable to hold this name:

```python
for _, msg, t in bag.read_messages(topics=['/sim_clock']):
```

Reply:
updated